File: Łatwe/C++/Imieniny/Imieniny.cpp
#include <iostream>

int main() {
    int D;
    std::cin >> D;
    for (int i = 0; i < D; i++) {
        int L;
        int C;
        std::cin >> L >> C;
        if (L > 1 && C % --L == 0)
            std::cout << "NIE\n";
        else
            std::cout << "TAK\n";
    }
}

File: Łatwe/Java/Imieniny/Imieniny.java
import java.util.Scanner;

class Imieniny {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        int D = in.nextInt();
        for (int i = 0; i < D; i++) {
            int L = in.nextInt();
            int C = in.nextInt();
            if (L > 1 && C % --L == 0)
                System.out.println("NIE");
            else
                System.out.println("TAK");
        }
    }
}
